src/opendap/coreServlet/ReqInfo.java

```diff
@@ -642,7 +642,7 @@ public static boolean isServiceOnlyRequest(HttpServletRequest req){
         return false;
     }
 
-    private static final String CF_History_Entry_Date_Format = "yyyy-MM-dd HH:mm:ss z";
+    private static final String CF_History_Entry_Date_Format = "yyyy-MM-dd'T'HH:mm:ss.S'Z'";
```

Comment:
I Am not convinced that this change is going to work for our historical CF history.  Please remove this change from this PR as it is not related to the ? and CE stuff and we'll discuss it later.